### PR TITLE
Update seattle temps data set with better sourced data

### PR DESCRIPTION
**Pull request by eitanlees**
I double checked and this won't affect any Vega-Lite examples. 

It should be a drop in replacement.

### CODE REVIEW

**Comment by domoritz** [comment]
Fixes #42

File: data/seattle-temps.csv [hunk 1]
```diff
@@ -1,8760 +1,8760 @@
...
-2010/04/29 07:00,47.8
-2010/04/29 08:00,49.5
-2010/04/29 09:00,51.3
-2010/04/29 10:00,53.1
-2010/04/29 11:00,54.8
-2010/04/29 12:00,56.3
-2010/04/29 13:00,57.4
-2010/04/29 14:00,58.2
-2010/04/29 15:00,58.5
-2010/04/29 16:00,58.
```
Comment (domoritz) [inline data/seattle-temps.csv]: I guess the completely correct format here would be `2010-12-31T23:00:00-08:00`, right? 

Unfortunately, the new format does not get parsed correctly in Safari. 

Screen Shot 2019-12-17 at 15 14 18

Comment (domoritz) [reply data/seattle-temps.csv]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Comment (eitanlees) [reply data/seattle-temps.csv]: good catch! Datetimes are always trickier than I expect. I will try to fix thing up tomorrow.

Comment (eitanlees) [reply data/seattle-temps.csv]: done!

**Pull request merged**